src/utils.js

```diff
@@ -73,3 +73,23 @@ export function listAllFields(rules) {
   });
   return allFields.flat();
 }
+
+export function removeFieldValue(path, formData = {}) {
+  let separator = path.indexOf(".");
```

Comment:
I think rjsf-conditionals uses the selectn library for similar path based lookups - is this path compatible and could we use that here to simplify this code?

Reply:
selectn library does not automatically look into nested array. For example,
```js
{a:{
 b:{
  c:[{d:2},{d:3}]
  }
}
}
```
To delete d using selectn, I need the path will be `a.b.c[0].d` and `a.c[1].d`. There is no way to know if the data is nested in array or if it is nested in multiple arrays